### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by desimone**
Prefer the shorter, more ubiquitous equivalent of `>/dev/null 2>&1` to  `1> /dev/null 2> /dev/null` .

Forgive my OCD twitch.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ ADD . /gopath/src/github.com/drone/drone/
 WORKDIR /gopath/src/github.com/drone/drone
 
 RUN apt-get update
-RUN apt-get -y install zip libsqlite3-dev sqlite3 1> /dev/null 2> /dev/null
+RUN apt-get -y install zip libsqlite3-dev sqlite3 >/dev/null 2>&1
```
Comment (dangra) [inline Dockerfile]: then what about using the even shorter syntax instead `&>/dev/null` 

Source from [bash manpage](http://www.gnu.org/software/bash/manual/bash.html#Redirecting-Standard-Output-and-Standard-Error)

> 3.6.4 Redirecting Standard Output and Standard Error
>
> This construct allows both the standard output (file descriptor 1) and the standard error output (file descriptor 2) to be redirected to the file whose name is the expansion of word.
>
> There are two formats for redirecting standard output and standard error:
>
>     &>word
>
> and
>
>     >&word
>
> Of the two forms, the first is preferred. This is semantically equivalent to
>
>     >>word 2>&1
>

Comment (desimone) [inline Dockerfile]: `&>/dev/null` is not portable, though that's probably irrelevant here.

Comment (dangra) [inline Dockerfile]: I think the real problem is that google/golang image is based on debian and it uses dash (not bash) as primary shell, so `&>/dev/null` won't work.

**Comment by dangra** [comment]
IMHO this is a pointless exercise as real development is happening on [branch 0.4.0](https://github.com/drone/drone/tree/0.4.0/) where [Dockerfile](https://github.com/drone/drone/blob/0.4.0/Dockerfile) changed a lot.

**Comment by desimone** [comment]
agreed.

**Comment by dangra** [comment]
:+1:

**Pull request closed without merging**